.github/workflows/release-pr.yml

```diff
@@ -1,88 +1,107 @@
 name: Create Release PR
 
 on:
-    workflow_dispatch:
-        inputs:
-            version:
-                description: "Release version"
-                required: true
-                default: 0.0.0
-                type: string
-
-    workflow_call:
-        inputs:
-            version:
-                default: 0.0.0
-                type: string
+  workflow_dispatch:
+    inputs:
```

Comment:
indentation changed from 4 to 2 spaces, which makes review harder.

The other workflows use 4 spaces indent so maybe change this back.